Gems/Atom/Feature/Common/Code/Source/CoreLights/PolygonLightFeatureProcessor.cpp

```diff
@@ -165,12 +178,8 @@ namespace AZ::Render
 
         auto transformedColor = AZ::RPI::TransformColor(lightRgbIntensity, AZ::RPI::ColorSpaceId::LinearSRGB, AZ::RPI::ColorSpaceId::ACEScg);
 
-        AZStd::array<float, 3>& rgbIntensity = m_polygonLightData.GetData<0>(handle.GetIndex()).m_rgbIntensityNits;
-
-        // Maintain sign bit in redsince it stores the convex / concave information of first two edges.
-        rgbIntensity[0] = copysignf(transformedColor.GetR(), rgbIntensity[0]);
-        rgbIntensity[1] = transformedColor.GetG();
-        rgbIntensity[2] = transformedColor.GetB();
+        AZStd::array<float, 3>& rgbIntensity = m_lightData.GetData<0>(handle.GetIndex()).m_rgbIntensityNits;
+        transformedColor.GetAsVector3().StoreToFloat3(rgbIntensity.data());
```

Comment:
So the sign isn't needed anymore? Just confirming.

Reply:
Correct, the sign was just a clever way to aid in calculating the direction vector in the shader from two edges, but now the direction is just provided as part of the light data.